### PR TITLE
251 save live video

### LINKED ISSUE #251: node that saves input.live into a recorded video after inferencing

### PR DESCRIPTION
**Pull request by ongzizhao**
To add functionality for live camera feed to save a video after model processing. Added filename and fps as outputs for live.py
filename = webcamfeed.mp4. fps = user input in live.yml.

Closes #251

### CODE REVIEW

File: peekingduck/pipeline/nodes/input/live.py [hunk 1]
```diff
@@ -47,7 +48,9 @@ def run(self, inputs: Dict[str, Any]) -> Dict[str, Any]:
                 img = resize_image(img,
                                    self.resize_info['width'],
                                    self.resize_info['height'])
-            outputs = {self.outputs[0]: img}
+            outputs = {self.outputs[0]: img, "fps":self.fps_saved_output_video,"filename":"webcamfeed.mp4"}
             return outputs
 
         raise Exception("An issue has been encountered reading the Image")
+
+
```
Comment (jwaycube) [inline peekingduck/pipeline/nodes/input/live.py:56]: The blank line is failing the linter `sh scripts/linter.sh`

File: peekingduck/pipeline/nodes/input/live.py [hunk 2]
```diff
@@ -47,7 +48,9 @@ def run(self, inputs: Dict[str, Any]) -> Dict[str, Any]:
                 img = resize_image(img,
                                    self.resize_info['width'],
                                    self.resize_info['height'])
-            outputs = {self.outputs[0]: img}
+            outputs = {self.outputs[0]: img, "fps":self.fps_saved_output_video,"filename":"webcamfeed.mp4"}
```
Comment (jwaycube) [inline peekingduck/pipeline/nodes/input/live.py:51]: It may be good to lift the filename as a config variable in live.yml .

**Review by jwaycube — CHANGES_REQUESTED** [review]

File: peekingduck/pipeline/nodes/input/live.py (hunk 1 above)
Comment (ongzizhao) [reply peekingduck/pipeline/nodes/input/live.py]: i only left 1 newline at the end.

File: peekingduck/pipeline/nodes/input/live.py (hunk 2 above)
Comment (ongzizhao) [reply peekingduck/pipeline/nodes/input/live.py]: i added the filename as a config variable in the latest commit

Comment (jwaycube) [reply peekingduck/pipeline/nodes/input/live.py]: Thanks

**Review by jwaycube — APPROVED** [review]

**Pull request merged**